[2020-10-29 00:28:15,790] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: weather-alert-source-data.print_start_time 2020-10-28T23:00:00+00:00 [queued]>
[2020-10-29 00:28:15,865] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: weather-alert-source-data.print_start_time 2020-10-28T23:00:00+00:00 [queued]>
[2020-10-29 00:28:15,866] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-10-29 00:28:15,866] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2020-10-29 00:28:15,866] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-10-29 00:28:15,905] {taskinstance.py:901} INFO - Executing <Task(BashOperator): print_start_time> on 2020-10-28T23:00:00+00:00
[2020-10-29 00:28:15,910] {standard_task_runner.py:54} INFO - Started process 373 to run task
[2020-10-29 00:28:16,230] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'weather-alert-source-data', 'print_start_time', '2020-10-28T23:00:00+00:00', '--job_id', '2', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/weather-alert-dag.py', '--cfg_path', '/tmp/tmpqzdwu8r2']
[2020-10-29 00:28:16,250] {standard_task_runner.py:78} INFO - Job 2: Subtask print_start_time
[2020-10-29 00:28:16,797] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: weather-alert-source-data.print_start_time 2020-10-28T23:00:00+00:00 [running]> c784450f7e27
[2020-10-29 00:28:17,023] {bash_operator.py:113} INFO - Tmp dir root location: 
 /tmp
[2020-10-29 00:28:17,057] {bash_operator.py:136} INFO - Temporary script location: /tmp/airflowtmpn_aed9dl/print_start_timebisnsi4b
[2020-10-29 00:28:17,084] {bash_operator.py:146} INFO - Running command: echo `date "+%Y-%m-%d%H:%M:%S"` "- Airflow Task Started"
[2020-10-29 00:28:17,163] {bash_operator.py:153} INFO - Output:
[2020-10-29 00:28:17,513] {bash_operator.py:157} INFO - 2020-10-2900:28:17 - Airflow Task Started
[2020-10-29 00:28:17,529] {bash_operator.py:161} INFO - Command exited with return code 0
[2020-10-29 00:28:17,773] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=weather-alert-source-data, task_id=print_start_time, execution_date=20201028T230000, start_date=20201029T002815, end_date=20201029T002817
[2020-10-29 00:28:21,134] {local_task_job.py:102} INFO - Task exited with return code 0
